Cargo.toml

```diff
@@ -16,18 +16,18 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-der = { version = "0.4", features = ["bigint", "derive"], optional = true }
-ecdsa = { version = "0.12", package = "ecdsa", default-features = false, optional = true }
-elliptic-curve = { version = "0.10", default-features = false, optional = true }
+der = { version = "0.6", features = ["derive"], optional = true }
```

Comment:
Thoughts on keeping these dependencies up to going forward?

For `-sys`, I released a `0.1.0` based on your changes, which I expect to be stable. But here, the core interest is in the Cortex M4 implementation, not so much keeping up with the "fallback".

On the other hand, if we add implementations for the `signature` crates (e.g. in particular `DigestSigner` with its more abstract `sign_digest` on top of our raw `sign_prehashed` - which I think we should keep), we might at least want to keep releasing breaking changes at least when `signature` or other trait crates have breaking changes.